R-package/configure

```diff
@@ -2101,8 +2101,6 @@ CXX14STD=`"${R_HOME}/bin/R" CMD config CXX14STD`
 CXX="${CXX14} ${CXX14STD}"
 CXXFLAGS=`"${R_HOME}/bin/R" CMD config CXXFLAGS`
 
-CC=`"${R_HOME}/bin/R" CMD config CC`
```

Comment:
We might need the CC flag here due to function registration in `R-package/src/init.c`.

Reply:
Yes, let's retain this line.